radixdlt-java/src/main/java/com/radixdlt/client/core/address/RadixUniverseConfigs.java

```diff
@@ -1,30 +1,27 @@
+	private RadixUniverseConfigs() {}
 
 	public static final RadixUniverseConfig getWinterfell() {
-		return RadixUniverseConfig.fromDson(testUniverseDson);
+		return RadixUniverseConfig.fromFile(getConfigFile("testuniverse.json"));
```

Comment:
get the stream instead of file
`return RadixUniverseConfig.fromInputStream(getConfigFileStream("testuniverse.json"));`